packages/devextreme-scss/scss/widgets/base/chat/layout/chat-messagebox/_mixins.scss

```diff
@@ -0,0 +1,34 @@
+@mixin chat-messagebox(
+  $padding,
+  $border-color,
+  $gap,
+  $textarea-min-height,
+  $textarea-border-radius,
+  $button-height,
+  $button-width,
+  $button-color
+) {
+  .dx-chat-message-box {
+    padding: $padding;
+    border-top: 1px solid $border-color;
+  }
+
+  .dx-chat-message-box-text-area {
+    &.dx-textarea.dx-texteditor {
+      min-height: $textarea-min-height;
```

Comment:
Let's discuss this privately

Reply:
Issues: 
1. The label is not center-aligned in Fluent
<img width="882" alt="image" src="https://github.com/user-attachments/assets/9bece778-234c-4b6c-b793-5cdbcaf1b990">
2. Height of input is 53px, not 48px
<img width="769" alt="image" src="https://github.com/user-attachments/assets/8b002b00-96b4-471a-87b2-e755e433b9db">

Currently, we are trying to set custom heights for text areas in themes that differ from the standard editor heights by ±2px. I don't understand why this is necessary, instead of adjusting the design of the text area to make it look like a 'textbox.'
@Shuvaevmedia